SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +56,8 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		Params.network.modular_kcp.security_type = GetDefault<USpatialGDKSettings>()->bUseSecureConnection ? WORKER_NETWORK_SECURITY_TYPE_TLS : WORKER_NETWORK_SECURITY_TYPE_INSECURE;
```

Comment:
Should this be DTLS? Is there an option for DTLS? I don't really know the difference but AFAIK the current default is DTLS. Will check

Reply:
It doesnt matter, DTLS and TLS are aliases and are chosen automatically based on the protocol.

https://improbable.slack.com/archives/CE5BYA3HD/p1581417230166600

Reply:
Yeah already checked that. ALL GRAVY BABY